packages/binding-websockets/package.json

```diff
@@ -26,9 +26,10 @@
     "typescript-standard": "0.3.36"
   },
   "dependencies": {
-    "@node-wot/td-tools": "0.7.2",
-    "@node-wot/core": "0.7.2",
     "@node-wot/binding-http": "0.7.2",
+    "@node-wot/core": "0.7.2",
+    "@node-wot/td-tools": "0.7.2",
+    "slugify": "^1.4.5",
```

Comment:
Please keep the original entries as is....

Reply:
Sorry, I just did `npm i slugify`, and apparently npm automatically sorted the dependencies.